module_evaluations/TCR-5_2021-12-08.MD

```diff
@@ -29,7 +29,7 @@ When performing a technical evaluation of a module, create a copy of this docume
 * [x] Sensitive information is not checked into git repository
 * [x] Module is written in a language and framework that FOLIO development teams are familiar with _e.g. Vertx/RMB, Spring Way/folio-spring-base, and React/Stripes_
 * [x] Back-end modules are based on Maven/JDK 11 and provide a Dockerfile
-* [ ] ~Integration (API) tests written in Karate if applicable -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_~
+* [x] ~Integration (API) tests written in Karate if applicable -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_~
```

Comment:
This seems to be changing the evaluation. How is this a typo?

Reply:
The typo is that I didn't follow the conventions of making something not applicable... it should be checked and struck-out.  I just struck it out, which is confusing.